landcover.mss

```diff
@@ -68,19 +64,15 @@
 @stadium: @leisure; // also sports_centre
 @golf_course: #b5e3b5;
 
+// --- Low-zoom ---
+
+@built-up-lowzoom: #d7d4d3;
+@vegetation-tall-lowzoom: #b6dca6;
```

Comment:
I don't know where exactly to put this comment, but forest color is still a blocker for me, and it will be worse if used for more object types. The contrast between this and vegetation low zoom is too strong. This alone makes more problems than using more than 4 colors on low zoom.

This proves to be a problem for both small dark patches of green (like Wales and Luxembourg z8+ on provided examples) and solid dark green areas (Shikoku on z8+).